html-aam/index.html

```diff
@@ -11625,6 +11625,87 @@ <h4 id="att-indeterminate">`indeterminate [IDL]`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="att-interestfor-attribute">`interestfor`</h4>
+        <table class="data" aria-labelledby="att-interestfor-attribute">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>
+                <span class="el-context"> `interestfor` </span>
+              </td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td>
+                <a data-cite="html/form-elements.html#the-button-element">`button`</a>;
+                <a data-cite="html/text-level-semantics.html#the-a-element">`a`</a>;
+                <a data-cite="html/image-maps.html#the-area-element">`area`</a>
+              </td>
```

Comment:
What is the reasoning for making the list of applicable elements so restricted? What about things like elements with `input` roles?

Reply:
this is called out in the intro to the explainer and there are links to related issues about this topic.
https://open-ui.org/components/interest-invokers.explainer/#introduction

tldr; these are the elements that had consensus for initial implementation, and further updates could come to extend to other elements.